syntax = "proto3";
package assistant_manager;

message SendTextRequest {
  int64 chat_id = 1;
  string text = 2;
}

message SendPhotoRequest {
  int64 chat_id = 1;
  string caption = 2;
  string base64_img = 3;
}

message MessageResponse {
  uint64 id = 1;
  int64 chat_id = 2;
}

service AssistantManager {
  rpc send_text(SendTextRequest) returns (MessageResponse);
  rpc send_photo(SendPhotoRequest) returns (MessageResponse);
}
